src/commands/support/reportedissues.ts

```diff
@@ -1,17 +1,86 @@
+import { Colors, EmbedField } from 'discord.js';
+import { JSDOM } from 'jsdom';
 import { CommandDefinition } from '../../lib/command';
 import { CommandCategory } from '../../constants';
 import { makeEmbed } from '../../lib/embed';
+import Logger from '../../lib/logger';
+
+const FBW_DOCS_REPORTED_ISSUES_URL = 'https://docs.flybywiresim.com/fbw-a32nx/support/reported-issues/';
+const FBW_DOCS_AUTOPILOT_ISSUES_URL = 'https://docs.flybywiresim.com/fbw-a32nx/feature-guides/autopilot-fbw/#typical-issues-and-how-to-solve-them';
+
+const genericReportedIssuesEmbed = makeEmbed({
+    title: 'FlyByWire A32NX | Reported Issues',
+    description: `Please see [this link](${FBW_DOCS_REPORTED_ISSUES_URL}) for a current list of reported issues.`,
+});
+
+const issueInSubsectionEmbed = (fields: EmbedField[]) => makeEmbed({
+    title: 'FlyByWire A32NX | Reported Issues',
+    description: 'Your issue is in our reported issues list. Please try the possible solutions in the following link(s) and report back if it didn\'t help, with all the steps you tried.',
+    fields,
+});
+
+const subsectionLinkEmbedField = (id: string): EmbedField[] => [
+    {
+        inline: false,
+        name: `${id.replace(/-/gi, ' ')}`,
+        value: `[link to reported issue section](${FBW_DOCS_REPORTED_ISSUES_URL}#${id})`,
+    },
+];
+
+const autopilotEmbed = makeEmbed({
+    title: 'FlyByWire A32NX | Reported Issues',
+    description: `Please see [this Link](${FBW_DOCS_AUTOPILOT_ISSUES_URL}) for typical issues with the custom autopilot and how to solve them.`,
+});
 
 export const reportedissues: CommandDefinition = {
     name: ['reportedissues', 'issues'],
     description: 'Provides a link to the reported issues page within docs',
     category: CommandCategory.SUPPORT,
-    executor: (msg) => {
-        const reportedissuesEmbed = makeEmbed({
-            title: 'FlyByWire A32NX | Reported Issues',
-            description: 'Please see [this link](https://docs.flybywiresim.com/reported-issues/) for a current list of reported issues.',
-        });
+    executor: async (msg) => {
+        try {
+            const args = msg.content.split(/\.|\n|\r/)
+                .at(1).toLowerCase()
+                .split(/\s+/)
+                .slice(1);
+
+            if (args === undefined || args.length === 0) {
```

Comment:
```suggestion
            if (args === undefined || args.length === 0 || args.at(0) === '') {
```
Fixes the bug if executing 
```
.issues .
``` 
or 
```
.issues<space>
test
```
(which that triggers an empty searchword, matching all the issues, overloading the embed)